cmd/compile/main.go

```diff
@@ -96,11 +128,18 @@ func run() error {
 		return err
 	}
 
+	if err := dockerRun("make", "INSTALL_MOD_PATH=modules_out", "modules_install", "-j"+strconv.Itoa(runtime.NumCPU())); err != nil {
+		return err
+	}
+
 	// copy and rename kernel
 	if err = sh.Copy(filepath.Join(dstFolder, "vmlinuz"), filepath.Join(bootFolder, "zImage")); err != nil {
 		return err
 	}
 
+	// copy modules
+	copy.Copy(filepath.Join(kernelFolder, "modules_out"), dstFolder)
+
```

Comment:
A `os.Rename` should be sufficient, no?

Reply:
Possibly. Only reason I opted for a copy is because every other operation done here is a copy as well. I can change that.

Reply:
I see 2 drawbacks to `copy.Copy`: 
- it adds a dependency
- if leaves the `linux-sources` git repository "unclean" (since `modules_out` is not in its `.gitignore`
=> using `os.Rename` solves both issues (but I would be fine with another solution which addresses these issues :)